### ISSUE TITLE
Test

### ISSUE THREAD

**Issue by JoCa96** [SchwarzIT/onyx#1913]

**Comment by Sinan-SchwarzIT** [comment]
@mj-hof When do you expect to finish the filter and search component? We have a go live in April and would like to use the filter component

**Comment by mj-hof** [comment]
Hi @Sinan-SchwarzIT. Thank you for your interest in this component. 

Basically, it's already possible to provide a filter and search functionality in your application by using the [select](https://storybook.onyx.schwarz/?path=/docs/form-elements-select--docs) and [input](https://storybook.onyx.schwarz/?path=/docs/form-elements-input--docs) components in combination. Get in touch with @larsrickert or @JoCa96 to get more infos to that. I hope this helps?!

**Comment by mj-hof** [comment]
> Yeah, for writing an own Filter component it we could do it with some selects and inputs, but it would be interesting if you plan to provide a complete component sometime in the near future

Yes, we do! We plan to implement a flexible component like the "filter bar smart" or "advanced search" available in CoreUI

**Comment by Sinan-SchwarzIT** [comment]
Awesome and do you have a rough timeline, when we could expect this flexible filter bar?
